app/models/edition_tag_form.rb

```diff
@@ -0,0 +1,27 @@
+class EditionTagForm
```

Comment:
Is it worth being explicit here by calling it `EditionTaxonomyTagForm` or something like that? (I'm fine either way)